api/unpack/index.go

```diff
@@ -18,9 +19,13 @@ import (
 // MountPath is where this handler is supposed to be mounted
 const MountPath = "/nix/store/"
 
-var nixCache = libstore.HTTPBinaryCacheStore{
-	CacheURI: getEnv("NAR_CACHE_URI", "https://cache.nixos.org"),
-}
+var ctx = context.Background()
+
+var nixCache, err = libstore.NewBinaryCacheReader(ctx, "s3://nar?region=eu-west-1&endpoint=http://127.0.0.1:9000&scheme=http")
+
+// libstore.HTTPBinaryCacheStore{
+// 	CacheURI: getEnv("NAR_CACHE_URI", "https://cache.nixos.org"),
+// }
 
 // Handler is the entry-point for @now/go as well as the stub main.go net/http
 func Handler(w http.ResponseWriter, req *http.Request) {
```

Comment:
Use the context of the request instead:
```suggestion
func Handler(w http.ResponseWriter, req *http.Request) {
  ctx := req.Context()
```